qtconsole/console_widget.py

```diff
@@ -432,7 +436,7 @@ def eventFilter(self, obj, event):
         # Make middle-click paste safe.
         elif getattr(event, 'button', False) and \
                 etype == QtCore.QEvent.MouseButtonRelease and \
-                event.button() == QtCore.Qt.MidButton and \
+                event.button() == (QtCore.Qt.MouseButton.MiddleButton if IsQt6 else QtCore.Qt.MidButton) and \
```

Comment:
Could you remove the continuation lines here and instead use parenthesis to wrap the code in this `elif`?

I'm referring to change this to

```python
elif (
    getattr(event, 'button', False) and
    etype == QtCore.QEvent.MouseButtonRelease and
    ...
):